### PR TITLE
Use oslo_config for config processing

### PR DESCRIPTION
**Pull request by GarySmith**
Define all config options, along with reasonable production defaults and
help text.
Move defaults.cfg to etc/devtest.conf.
Support the use of oslo-config-generator in tox (tox -e genconfig) in
order to create a sample, documented config file.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -46,5 +41,5 @@ supported:
        Retrieves the details of the given server from HPE OpenView
 
   ``/api/v1/clm/...``
-       Requests to this URL are forwarded to the correspondingin 
+       Requests to this URL are forwarded to the correspondingin
```
Comment (jack-likes-to-code) [inline README.rst]: corresponding in

File: README.rst [hunk 2]
```diff
@@ -46,5 +41,5 @@ supported:
        Retrieves the details of the given server from HPE OpenView
```
Comment (jack-likes-to-code) [inline README.rst]: All the OpenViews in this file should be OneView

File: ardana_installer_server/main.py [hunk 3]
```diff
@@ -29,13 +30,18 @@
 try:
     logging.basicConfig(level=logging.DEBUG,
                         filename='/var/log/cloudinstaller/install.log')
```
Comment (jack-likes-to-code) [inline ardana_installer_server/main.py]: Can we put the log file into the config file, too?

File: ardana_installer_server/main.py [hunk 4]
```diff
@@ -29,13 +30,18 @@
...
-except IOError as e:
+except IOError:
```
Comment (jack-likes-to-code) [inline ardana_installer_server/main.py]: Can we put the log file into the config, too?

Comment (GarySmith) [reply ardana_installer_server/main.py]: Well, the right way to do that would be to introduce oslo logging, which I was intending to do next, in both the shim and the ardana_service. oslo logging has a bunch of configurable parameters that we get for free, including this one

**Pull request merged**